README.md

```diff
@@ -59,7 +59,9 @@ This makes riding horses around path blocks way less frustrating.
 <details><summary> Allow swimming while Mounted </summary>
 
 Normally Horses will sink in water when you are riding them.\
-This makes you float on the surface as if they weren't mounted
+This makes you float on the surface as if they weren't mounted.\
+The default configs turn this ability off for skeleton and zombie horses, and camels.\

```

Comment:
Change to 
> The default config turns this ability off for skeleton and zombie horses, and camels.\

Reply:
> Change to
> 
> > The default config turns this ability off for skeleton and zombie horses, and camels.\

This is already in the new readme in my fork;
```md
<details><summary> Allow swimming while Mounted </summary>

Normally Horses will sink in water when you are riding them.\
This makes you float on the surface as if they weren't mounted.\
The default configs turn this ability off for skeleton and zombie horses, and camels.\
Have you ever seen a camel swim? I think not!

</details>
```
Maybe merge this too?

Reply:
Wait never mind, I read it wrong, fixed sorry.